ext/mysqlnd/mysqlnd_structs.h

```diff
@@ -231,7 +231,7 @@ typedef struct st_mysqlnd_session_options
 
 	char 		*charset_name;
 	/* maximum allowed packet size for communication */
-	ulong		max_allowed_packet;
+	size_t		max_allowed_packet;
```

Comment:
After checking the source code, apparently this is an `unsigned int`: https://github.com/php/php-src/blob/master/ext/mysqlnd/mysqlnd_connection.c#L1729

Reply:
Fixed. Yes, I think this should be correct... :D thanks.

Reply:
There is however this:
https://github.com/php/php-src/blob/master/ext/mysqlnd/mysqlnd_auth.c#L290
where the max_packet_size is uint32_t...

Reply:
Luckily `unsigned int` and `uint32_t` are very likely to be the same type :)